armory/utils/config_loading.py

```diff
@@ -30,7 +30,7 @@
 from armory.art_experimental.attacks import patch
 from armory.data.datasets import ArmoryDataGenerator, EvalGenerator
 from armory.data.utils import maybe_download_weights_from_s3
-from armory.utils import labels
+from armory.utils import label_targeters
```

Comment:
I would prefer to keep the original "labels" module name.

Reply:
switched it back